lib/models/asset-size-printer.js

```diff
@@ -63,16 +63,16 @@ module.exports = class AssetPrinterSize {
         })
         // Print human-readable file sizes (including gzipped)
         .map((file) => {
-          let contentsBuffer = fs.readFileSync(file);
-          return gzip(contentsBuffer).then((buffer) => ({
-            name: file,
-            size: contentsBuffer.length,
-            gzipSize: buffer.length,
-            showGzipped: contentsBuffer.length > 0,
-          }));
+          return pool
+            .exec('gzipStats', [file])
+            .then((result) => result)
+            .catch((err) => console.error(err));
```

Comment:
should we be handling error like this ? `console.error` was not there before

Reply:
I agree we shouldn't be handling error like this, any pointers?